libs/common/src/helpers/is-gef-facility-type.test.ts

```diff
@@ -0,0 +1,19 @@
+import { isGefFacilityType } from './is-gef-facility-type';
+import { FACILITY_TYPE } from '../constants';
+
+describe('is-gef-facility-type helper', () => {
+  it('should return true when type is `CASH`', () => {
+    const result = isGefFacilityType(FACILITY_TYPE.CASH);
+    expect(result).toEqual(true);
+  });
+
+  it('should return true when type is `CONTINGENT`', () => {
+    const result = isGefFacilityType(FACILITY_TYPE.CONTINGENT);
+    expect(result).toEqual(true);
+  });
+
+  it('should return false when type is NOT a GEF facility type', () => {
```

Comment:
same as above

Reply:
Refactored, this file is no longer changed from before